sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -80,7 +80,7 @@ private[sql] class SessionState(ctx: SQLContext) {
   /**
    * Parser that extracts expressions, plans, table identifiers etc. from SQL texts.
    */
-  lazy val sqlParser: ParserInterface = new SparkQl(conf)
+  lazy val sqlParser: ParserInterface = SparkSqlParser
```

Comment:
It is still in use in Hive. I'll open up a follow-up PR in the following week or so which will also parse the Hive commands.